node/src/main/java/io/airlift/node/NodeInfo.java

```diff
@@ -261,7 +261,8 @@ private static InetAddress findPublicIp()
         InetAddress localAddress = null;
         try {
             localAddress = InetAddress.getLocalHost();
-            if (isV4Address(localAddress) && getGoodAddresses().contains(localAddress)) {
+            if ((isV4Address(localAddress) || isV6Address(localAddress)) 
```

Comment:
I don't think we want this, since we prefer to choose an IPv4 address if it exists. If we do want this, these checks can be removed, and the comments and documentation for this class need to be updated.